gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -77,6 +78,10 @@ public class AndroidCommandPluginExtension {
         System.properties['seed'] ?: seed
     }
 
+    def getCategories() {
+        System.properties['categories'] ?: categories ?: ''
```

Comment:
I believe there are three possible, scenarios here. 
Highest priority is to `System.properties[categories]`, then specified categories `categoires` in gradle file and then `default` one. 

I copied this bit from getEvents :arrow_up: and this is how I think it works